apps/studio/src/components/PageEditor/LinkEditorModal.tsx

```diff
@@ -72,7 +80,7 @@ interface LinkEditorModalContentProps {
   linkText?: string
   linkHref?: string
   onSave: (linkText: string, linkHref: string) => void
-  linkTypes: LinkTypeMapping
+  linkTypes: LinkEditorModalProps["linkTypes"]
```

Comment:
why the change?

Reply:
the child component should depend on the parent rather than specifying an external dependency.

actually i'l just use `Extract` here rather than respecificying oops